src/ChartInternal/ChartInternal.ts

```diff
@@ -284,9 +285,9 @@ export default class ChartInternal {
 			$$.point = $$.generatePoint();
 		}
 
-		if (state.hasAxis) {
-			$$.initClip();
+		$$.initClip();
```

Comment:
This breaks tree-shaken builds because `initClip` is not defined at this point. The clip internals are loaded when loading the axis feature. they are not imported in this file.